### DISCUSSION TITLE
Study Group

### DISCUSSION THREAD

**Discussion by ataluzz** [microsoft/ML-For-Beginners#170] [Q&A]
Hi everyone, 

I've been wanting to study ML for a long time now, but sometimes I really lack motivation. So I thought about creating a group of people, who also just started to study ML. We can support each other :) 
If you are interested and have any ideas about format/platform for this group, write in this discussion. 
P.S. Sorry if there are any mistakes in my message, I'm not from England or America :)

**Comment by jlooper** [comment]
A very nice idea! Let's gather!

**Comment by Bildad0** [comment]
I think a telegram group is convenient.

**Reply by jlooper** [reply]
I’d prefer to keep things centralized here so I can assist if needed but of course up to you all!

**Reply by parisa-ahmadi** [reply]
Hi dear, happy to see you here 🌹 how can start to learn this course here, I'm beginner I have full of potential to learn something new specifically in data science, Ai and Ml. I don't know how to start and work with github to use this course I'm new member of github

**Reply by jlooper** [reply]
hi, this is a self-study course. You can ask questions here on the discussion boards. To interact with the course, fork the repo and download it to your local computer, or zip and download the files. Open the folders in Visual Studio Code and run the notebooks locally. There are instructions in the 'tooling' folder including a video on how to set up your system.

**Comment by Yvhong123** [comment]
How can I join you? Is there any QQ or Wechat group?

**Reply by jlooper** [reply]
You can work through the lessons here and start discussion threads, if that seems interesting.

**Comment by pavittarx** [comment]
For me, I only started learning ML, recently, So, I'd be glad to find someone who may want to walk through together. The story is same for me, but I am trying to walk through as when I get time and things make sense to me.

**Reply by rabichubu** [reply]
How can i join ypu

**Comment by jlooper** [comment]
What I hear from your feedback is that we ought to have a Study Jam, like the GDGs do: https://events.withgoogle.com/cloud-studyjam/

Let's think about how we can do this while respecting each others' time zones and time commitments. I can certainly support this effort.

**Reply by pavittarx** [reply]
If we are to do a study Jam. Here are my suggestions for the same: 
- A discussion thread or chat room on Discord / Gitter / Telegram for keeping in touch about the planning or discussing any upcoming sessions. 
- We can plan according to time zones and days we are or will be available. Say I have `IST` timezone, and I can preferably be available at XX hours during this week on these days. There can be other ways going around with this.
- A public discussion board, so that we can announce what comes next, and if someone new comes in they can join these groups as well. 

There would still be more to this, but I suppose this can help us as a starter.

**Reply by jlooper** [reply]
This makes sense, plus I think we need some kind of 'office hours' of people in my organization, to cover certain topics. Let me check to see if the Microsoft Reactor team can help us with some of these logistics. I'll see about Discord as well, we might already have something set up.

thanks!
Jen

**Reply by pavittarx** [reply]
I'd be glad to join if something already is there. However, please do let me know if there is some update on this or if I could be helpful in any way possible. 

Also, thanks for all your support.

**Comment by jlooper** [comment]
Update: I have access to a Python Discord server that we are involved in and we could probably create a Study Jam channel. I'm going to ask for support from our Reactors (our locally-based event spaces) for streaming, so stay tuned please! :)

**Comment by ataluzz** [comment]
Didn't think there will be such support from anybody other than students. 
@jlooper, thank you for your help! It will be great if the Discord channel would work out :)

**Comment by pavittarx** [comment]
thanks there @jlooper. I'm waiting for updates to come. :) 

@ataluzz thanks for starting this thread.

**Comment by jlooper** [comment]
hi, a quick update. I have two ideas:

1. We can gather at our Python Discord room
2. Student Ambassadors can lead the study group
cc @Rishit-dagli do you think you could lead this initiative?

**Reply by pavittarx** [reply]
Both or either of them would be helpful. :)

**Comment by jlooper** [comment]
https://github.com/microsoft/ML-For-Beginners/discussions/276

**Comment by Shujasheikh** [comment]
No problem will motivate you, your on right platform to learn ML system and your English is good enough to start discussion secondly every one is learning 👍

**Comment by Shujasheikh** [comment]
I can't walk but I can share my ideas with you.

**Comment by Shujasheikh** [comment]
Hi, there

Thanks for your interest into github plus ML system, github is a platform
to learn share your knowledge as you mentioned your beginner of ML system
and want learn in detail do join course at github ask questions in related
to ML and options available there.

Hope you will find helpful.

Thanks & Best Regards
SHEIKH

**Comment by Akela-Wan** [comment]
Can I join you, my friend

**Comment by mubarak-mustopha** [comment]
I always get stuck whenever I'm trying do some data analysis before actually building a model.
Like I don't really know the right questions to ask based on the data I have.....can anyone make a recommendation or a resource that can help me solve this problem?

**Comment by Sanjaypranav** [comment]
Mostly everyone are very passionate about learning :+1: here i can give some tips 
1) Reading over watching - read more blogs instead of watching more videos that gives you creative thinking 
2) Practice with dataset 
3) talk with experts how to handle new problems using machine learning 
4) Do research or read research paper
